src/sentry/api/endpoints/organization_metrics_tag_details.py

```diff
@@ -35,6 +36,28 @@ def get(self, request: Request, organization: Organization, tag_name: str) -> Re
                 {"detail": "You must supply at least one project to see its metrics"}, status=404
             )
 
+        if all(
+            features.has("projects:use-eap-spans-for-metrics-explorer", project)
+            for project in projects
+        ):
+            if len(metric_names) == 1 and metric_names[0].startswith("d:eap"):
```

Comment:
Couldn't this be fetched from the metrics platform still? At least then we'd get (more) realistic data.